routers/bpa_register.py

```diff
@@ -27,57 +27,29 @@
 )
 
 
-def send_approval_email(registration: BPARegistrationRequest, bpa_resources: list[Resource]):
+def send_approval_email(registration: BPARegistrationRequest):
```

Comment:
Do we need this approval email? BPA access should be automatic

Reply:
Do we have that in any of the BPA user stories? If not I think we should remove.